### PR TITLE
Keep identifier option in anonymous mode

### PR DESCRIPTION
**Pull request by imnutz**

### CODE REVIEW

File: lib/utils/misc.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+var SSC_COOKIE_NAME = '_td_ssc_id'
```
Comment (tung-vu-td) [inline lib/utils/misc.js:1]: There are many constants in td-js-sdk which, IMHO, should be collected to utils/constants.js file.

This is just a suggestion going forward. it's up to you whether to do it in this PR or not. I understand this PR is urgent :)

File: lib/record.js [hunk 2]
```diff
@@ -85,16 +87,29 @@ exports.setSignedMode = function setSignedMode () {
...
   if (this.client.storeConsentByLocalStorage) {
     global.localStorage.setItem(SIGNEDMODECOOKIE, 'false')
   } else {
     setCookie(this.client.storage, SIGNEDMODECOOKIE, 'false')
   }
 
-  this.resetUUID(this.client.storage, this.client.track.uuid)
-  // remove global cookie too
-  cookie.removeItem(this.client.globalIdCookie)
+  if (!keepIdentifier) {
```
Comment (tung-vu-td) [inline lib/record.js:97]: Why does this not follow the decentralized way currently employed by the SDK. For example: `setCookie(this.client.storage, NONSIGNEDMODEKEEPIDENTIFER, 'true')`. I'm not sure I understand the need to centralized cookie logic for cookie here in record.js

IMO, unless I'm missing some important info here, each logic should be moved back to its original plugin, with extra check of NONSIGNEDMODEKEEPIDENTIFER

- [ ] Review this logic

**Review by tung-vu-td — CHANGES_REQUESTED** [review]

File: lib/record.js (hunk 2 above)
Comment (imnutz) [reply lib/record.js:97]: With new cookie, it requires refreshing the page to apply the logic which might not be what we want.

**Review by tung-vu-td — APPROVED** [review]

**Review by flexgrip — APPROVED** [review]
Reviewed for data exfiltrations.

**Pull request merged**